examples/automatic_folder/versions.tf

```diff
@@ -23,7 +23,7 @@ terraform {
     }
     google = {
       source  = "hashicorp/google"
-      version = "~> 4.0"
+      version = "< 5.0"
```

Comment:
Since the modules in this example are pinned to a version, we can continue to use `~> 3.x`